app/views/api/v1/spots/create.json.jbuilder

```diff
@@ -1,3 +1,5 @@
 json.spot do
   json.partial! 'info', spot: @spot
 end
+
+json.matches @spot.matches, partial: 'match_info', as: :match unless @spot.matches.count.zero?
```

Comment:
Since you are adding `has_many :matches` to `Spot` model you will have access to the matches for a given spot.

`json.matches @spot.matches, partial: 'match_info', as: :match if @spot.matches`